src/main/java/mate/academy/dao/Dao.java

```diff
@@ -1,8 +1,11 @@
-package mate.academy.lib;
+package mate.academy.dao;
```

Comment:
why do you need to update package?

Reply:
I just wanted all the dao-related stuff to be in one package. That was basically only for me to clearly see, they're related. I can remove that, if such naming is bad practice

Reply:
@AlexRosenkreuz yes, pls don't do irrelevant changes in your PRs

you should add only relevant changes